programs/drift/src/state/order_params.rs

```diff
@@ -92,6 +92,60 @@ impl OrderParams {
 
         Ok(())
     }
+
+    pub fn get_aggressive_close_params(
+        market: &PerpMarket,
+        direction_to_close: PositionDirection,
+        base_asset_amount: u64,
+    ) -> DriftResult<OrderParams> {
+        let estimated_mark_offset = market
+            .amm
+            .last_ask_price_twap
+            .safe_add(market.amm.last_bid_price_twap)?
```

Comment:
should this max with 0?

Reply:
think you want to start at the middle (mark) of these and end at the opposite side + buffer (based on volatility)